db/limitersMgr.go

```diff
@@ -0,0 +1,696 @@
+package db
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/distributedio/titan/conf"
+	"github.com/distributedio/titan/metrics"
+	sdk_kv "github.com/pingcap/tidb/kv"
+	"go.uber.org/zap"
+	"golang.org/x/time/rate"
+	"net"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+)
+
+const (
+	LIMITDATA_DBID             = 0
+	ALL_NAMESPACE              = "*"
+	NAMESPACE_COMMAND_TOKEN    = "@"
+	QPS_PREFIX                 = "qps:"
+	RATE_PREFIX                = "rate:"
+	LIMIT_VALUE_TOKEN          = " "
+	LIMITER_STATUS_PREFIX      = "limiter_status:"
+	LIMITER_STATUS_VALUE_TOKEN = ","
+	TIME_FORMAT                = "2006-01-02 15:04:05"
+	MAXIMUM_WEIGHT             = 1
+	MINIMUM_WEIGHT             = 0.1
+)
+
+type LimiterWrapper struct {
+	limiterName  string
+	limiter      *rate.Limiter
+	globalLimit  int64
+	localPercent float64
+	lock         sync.Mutex
+}
+
+type CommandLimiter struct {
+	localIp     string
+	limiterName string
+
+	qpsLw  LimiterWrapper
+	rateLw LimiterWrapper
```

Comment:
QPS has the same meaning of `request rate`, we should use more friendly names as the limitation type like `Command` and `DataFlow`